server/rosetta/client_online.go

```diff
@@ -105,16 +141,14 @@ func (c *Client) accountInfo(ctx context.Context, addr string, height *int64) (a
 		return nil, crgerrs.FromGRPCToRosettaError(err)
 	}
 
-	var account auth.AccountI
-	err = c.ir.UnpackAny(accountInfo.Account, &account)
+	signerData, err := c.converter.ToRosetta().SignerData(accountInfo.Account)
 	if err != nil {
-		return nil, crgerrs.WrapError(crgerrs.ErrCodec, err.Error())
+		return nil, err
 	}
-
-	return account, nil
+	return signerData, nil
 }
 
-func (c *Client) Balances(ctx context.Context, addr string, height *int64) ([]*types.Amount, error) {
+func (c *Client) Balances(ctx context.Context, addr string, height *int64) ([]*rosettatypes.Amount, error) {
```

Comment:
godoc is missing

Reply:
godocs are missing because those methods are part of an interface implementation - which is in itself throughly documented - I will add the comments that those methods implement the said interface